src/core.js

```diff
@@ -284,6 +284,16 @@ function done() {
 		total: config.stats.all,
 		runtime: runtime
 	} );
+
+	// Clear own sessionStorage items if all tests passed
+	if ( config.reorder && defined.sessionStorage && config.stats.bad === 0 ) {
+		for ( i = 0; i < sessionStorage.length; i++ ) {
+			key = sessionStorage.key( i++ );
```

Comment:
this for loop can finally be replaced by forEach or even better `[].reduce`.

> Removing items from an object while iterating over it also seems fishy.

Yes, it's fishy. But this PR is just moving a pre-existent code, so I believe we can address it later.

@trentmwillis I want to confirm you agree, in this case can you open an issue to keep track of this?